tree/ntuple/v7/src/RNTupleDescriptorFmt.cxx

```diff
@@ -110,6 +110,9 @@ void ROOT::Experimental::RNTupleDescriptor::PrintInfo(std::ostream &output) cons
       info.fElementSize = elementSize;
       info.fType = column.second.GetModel().GetType();
 
+      if (column.second.IsAliasColumn())
+         continue;
```

Comment:
It is not straightforward to understand why we skip alias columns.

Reply:
Addressed in #13589 which is underlying this PR.